clients/admin-ui/src/features/user-management/DeleteUserModal.tsx

```diff
@@ -32,19 +32,18 @@ function DeleteUserModal(user: User) {
       setConfirmValue(event.target.value);
     }
   };
+  const {id: userId, username} = user;
 
   const deletionValidation =
-    user.id &&
+    !!(userId &&
```

Comment:
Could this just be `userId`? Is there a need for the double !!

Reply:
hm I honestly can't remember why I changed this 😅  the `!!` forces the type (which can be "" or undefined) to a boolean which can be useful, but doesn't look like it's required in this case